src/dag/meta.fbs

```diff
@@ -0,0 +1,9 @@
+namespace meta;
+
+// Metadata about a chunk, stored separately to enable exploring
```

Comment:
Can you modify this comment to explain what the refs are?

Reply:
Done.